deployment/digital-ocean/README.md

```diff
@@ -24,3 +24,15 @@ Digital Ocean kubernetes clusters don't have a graphical interface, so I suggest
 to setup the [kubernetes dashboard](./dashboard/README.md) as a next step.
 Configuring [HTTPS](./https/README.md) is bit tricky and therefore I suggest to
 do this as a last step.
+
+## Spaces
+
+We are storing our images in the s3-compatible [DigitalOcean Spaces](https://www.digitalocean.com/docs/spaces/). 
+
+We still want to take backups of our images in case something happens to the images in the cloud. See these [instructions](https://www.digitalocean.com/docs/spaces/resources/s3cmd-usage/) about getting set up with `s3cmd` to take a copy of all images in a `Spaces` namespace, ie `human-connection-uploads`.
+
+After configuring `s3cmd` with your credentials, etc. you should be able to make a backup with this command.
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-04-06T13:25:38Z" title="Monday, April 6th 2020, 3:25:38 pm +02:00">Apr 6, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,3 +24,15 @@ Digital Ocean kubernetes clusters don't have a graphical interface, so I suggest
 to setup the [kubernetes dashboard](./dashboard/README.md) as a next step.
 Configuring [HTTPS](./https/README.md) is bit tricky and therefore I suggest to
 do this as a last step.
+
+## Spaces
+
+We are storing our images in the s3-compatible [DigitalOcean Spaces](https://www.digitalocean.com/docs/spaces/). 
+
+We still want to take backups of our images in case something happens to the images in the cloud. See these [instructions](https://www.digitalocean.com/docs/spaces/resources/s3cmd-usage/) about getting set up with `s3cmd` to take a copy of all images in a `Spaces` namespace, ie `human-connection-uploads`.
+
+After configuring `s3cmd` with your credentials, etc. I was able to make a backup with this command.
```


"I"? Hope there is more than one person writing the docs :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-06T16:31:50Z" title="Monday, April 6th 2020, 6:31:50 pm +02:00">Apr 6, 2020</time>_

---


that is a good point...